docs/en/sql-reference/aggregate-functions/reference/intervalLengthSum.md

```diff
@@ -0,0 +1,103 @@
+---
+toc_priority: 146
+toc_title: intervalLengthSum
+---
+
+# intervalLengthSum {#agg_function-intervallengthsum}
+
+Calculates the sum of the length of all ranges (segments on numeric axis) excluding intersections.
```

Comment:
'excluding intersections' sounds like we do not count intersections at all, but we count it once
```suggestion
Calculates the total length of union of all ranges (segments on numeric axis).
```